src/scripts.js

```diff
@@ -16,7 +16,7 @@ import Sleep from './Sleep';
 
 
 let todayDate = "2019/09/22";
-
+let hyData = hydrationData
```

Comment:
not sure what this variable is used for?

Reply:
This was just a test for an issues I found that I forgot to remove

Reply:
Looks like I did actually remove it